apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -1408,6 +1408,26 @@
         "title": "Stake",
         "description": "Grow your crypto with Ledger"
       }
+    },
+    "recoverBanner": {
+      "subscribeDone": {
+        "title": "Add your payment card",
+        "description":"Add your payment method to get your backup secure.",
+        "primaryCta": "Add now",
+        "secondaryCta": "Maybe later"
+      },
+      "verifyIdentity": {
+        "title": "Verify your identity",
+        "description":"Verify your identity to get your backup secure.",
+        "primaryCta": "Verify now",
+        "secondaryCta": "Maybe later"
+      },
+      "connectDevice": {
+        "title": "Connect your Ledger",
+        "description":"Connect your Ledger to get your backup secure.",
+        "primaryCta": "Connecting my Ledger",
```

Comment:
"Connect my Ledger" ?

Reply:
yes you are right, I will fix that quickly

Reply:
https://github.com/LedgerHQ/ledger-live/pull/5453/commits/a8ea71743b826139018528d73e210fc155163da3